### ISSUE TITLE
Add labels to repository

### ISSUE THREAD

**Issue by johlju** [dsccommunity/dsccommunity.org#10]
Suggest adding labels to this repo to track issues and PR's. This list of labels are from the DscResource repo with some removed that is not applicable to this repo.

Example on these labels can be seen here https://github.com/PowerShell/SqlServerDsc/issues/labels

Label | Description | Color Hex | Area of usage (PR, Issue) | Comment
-- | -- | -- | -- | --
bug | The issue is a bug. | #ee0701 Red | Issue |  
discussion | The issue is a discussion. | #993399 Purple | Issue |  
documentation | The issue is related to documentation only. | #c5def5 Lighter blue | Issue |
duplicate | The issue or PR is the duplicate of another. | #ffffff White | Issue, PR | Indicates reason for closure. Maintainer needs to comment why the issue/PR was closed.
good first issue | The issue should be easier to fix and can be taken up by a beginner to learn to contribute on GitHub | #5319e7 Purple | Issue | The issue should be easier to fix and can be taken up by a beginner who wants to learn the process of contribute on GitHub. GitHub Help point out to use this and the ‘Help wanted’ tag, see [Helping new contributors find your project with labels](https://help.github.com/articles/helping-new-contributors-find-your-project-with-labels). ***Note:** The label description is missing punctuation because of length restriction in GitHub.*
enhancement | The issue is an enhancement request. | #84b6eb Light blue | Issue |  
external | The issue cannot be resolved within the DSC Resource Kit. | #ffffff White | Issue | Indicates reason for closure. Maintainer needs to comment why the issue/PR was closed.
help wanted | The issue is up for grabs for anyone in the community. | #128A0C Green | Issue |  
high priority | The issue or PR should be resolved first. It is of less priority than the label 'Blocking Release'. | #ffcc99 Light orange | Issue, PR | The issue or pull request is of high importance and should be resolved or merged first, but it is of less priority than anything with the label 'Blocking Release'.
in progress | The issue is being actively worked on by someone. | #99ffcc Turquoise | Issue |  
needs investigation | The issue needs to be investigated by the maintainers or/and the community. | #ffff99 Yellow | Issue | The issue needs to be investigated by the maintainers or/and the community to find the cause of a problem, the fix for a problem, the benefit of a request, or the answer to a question.
needs more information | The issue needs more information from the author or the community. | #ffff99 Yellow | Issue |
needs review | The pull request needs a code review. | #99ff33 Lime green | PR |  
not fixed | The issue was closed without being fixed. | #ffffff White | Issue | Indicates reason for closure. Maintainer needs to comment why the issue/PR was closed.
on hold | The issue or pull request has been put on hold by a maintainer. | #1e1e7b Dark blue | Issue, PR | Maintainer needs to comment why the issue/PR is on hold. The issue/PR cannot be closed or merged until the label is removed with the consent of whoever assigned the label.
question | The issue is a question. | #cc317c Dark pink | Issue |  
ready for merge | The pull request was approved by the community and is ready to be merged by a maintainer. | #215e82 Dark cyan | PR |
waiting for author response | The pull request is waiting for the author to respond to comments in the pull request. | #ffff99 Yellow | PR | Should *never* be use together with 'Waiting for code fix'.
waiting for CLA pass | The contributor has not yet signed the CLA so that pull request check is not successful. | #ee0701 Red | PR |  
waiting for code fix | A review left open comments, and the pull request is waiting for changes to be pushed by the author. | #ffff99 Yellow | PR | A review of the pull request left open comments, and the pull request is waiting for new commits to be push to the pull request by the author. This label takes precedence over 'Waiting for author response'. This label should *never* be use together with 'Waiting for author response'.

**Comment by johlju** [comment]
Label | Description | Color Hex | Example
-- | -- | -- | -- 
RFC - Committee Reviewed | RFC has been reviewed. | #128A0C Green | ![image](https://user-images.githubusercontent.com/7189721/61530872-97467600-aa25-11e9-8af0-d609adb4150d.png)
RFC - Waiting on Committee Review | RFC has been closed for comments and are waiting on committee review. | #fbca04 Dark Yellow | ![image](https://user-images.githubusercontent.com/7189721/61531317-c8737600-aa26-11e9-8e69-67a038412383.png)
RFC - Open for Comments | RFC is accepting comments from the community. | #e3f1dA Lighter Green | ![image](https://user-images.githubusercontent.com/7189721/61531367-eccf5280-aa26-11e9-99c8-b34f0f22bacb.png)
RFC - Waiting on Author | RFC is waiting to be updated by the author. | #ffff99 Yellow | ![image](https://user-images.githubusercontent.com/7189721/61531420-08d2f400-aa27-11e9-95fd-6bf22df6c8d4.png)

**Comment by gaelcolas** [comment]
Should the RFC - Committee Reviewed be split in 2 items? Approved (green) and Rejected (red)?
I'd try to shorten the text:
- [RFC] Approved
- [RFC] Declined
- [RFC] To Review
- [RFC] Draft
- [RFC] Waiting on Author

**Comment by johlju** [comment]
That works, I used the names from the PowerShell RFC repo. Shoudl we skip "Open for Comments" or is that "Draft"?

**Comment by johlju** [comment]
Label | Description | Color Hex | Example
-- | -- | -- | -- 
[RFC] Approved | RFC has been reviewed and has been approved. | #128A0C Green | ![image](https://user-images.githubusercontent.com/7189721/61545614-22d0fe80-aa48-11e9-99c9-c3e7b636adb3.png)
[RFC] Declined | RFC has been reviewed and has been declined. | #d73a4a Red | ![image](https://user-images.githubusercontent.com/7189721/61545521-f321f680-aa47-11e9-966c-ac8da24ffc3b.png)
[RFC] To Review | RFC has been closed for comments and are waiting on committee review. | #fbca04 Dark Yellow | ![image](https://user-images.githubusercontent.com/7189721/61545665-45631780-aa48-11e9-98de-8f1e9e8d98a2.png)
[RFC] Draft | RFC is accepting comments from the community. | #e3f1da Lighter Green | ![image](https://user-images.githubusercontent.com/7189721/61545892-c4585000-aa48-11e9-9369-b878c1e0b8bf.png)
[RFC] Waiting on Author | RFC is waiting to be updated by the author. | #ffff99 Yellow | ![image](https://user-images.githubusercontent.com/7189721/61546008-fa95cf80-aa48-11e9-91bc-2e2c29c29292.png)

Might be that it looks better with upper-case `[RFC]` and just lower-case for the other text. 🤔

**Comment by gaelcolas** [comment]
Yeah, anything that's a draft means it's open for comment. When we get to "To Review", that means it's mature enough for the committee to review (although discussion can still happen, obviously).

Up to you for the case, it needs to be functional, I don't really care about design 😛

**Comment by johlju** [comment]
Looking at the folders in the repo https://github.com/PowerShell/PowerShell-RFC I thinking we need a label for `[RFC] Withdrawn` so we code label closed PR's/Issues that reason as well. 🤔  Adding and see if we agree to that, or come up with some other methods tracking that.

**Comment by gaelcolas** [comment]
happy with that. Feel free to create them, but no rush.

**Comment by johlju** [comment]
I have a script that creates them, so it was fixed in a couple of minutes after adding the new ones.
